config.yaml

```diff
@@ -2,6 +2,12 @@ serverConfig:
   port: 8080
 cloudEventServerConfig:
   port: 8181
+jobConfig:
+  lockName: horizon-lock
```

Comment:
Can this be set in the code? What's the best practice about this? How does other projects like kubernetes do  this?